docs/scos/dev/front-end-development/migration-guide-remove-support-of-ie11.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Migration guide - Remove support of IE11
+description: Use this guide to migrate the project from Yarn to NPM v8.
+template: concept-topic-template
+---
+
+This document provides instructions for removing support of IE11 in your Spryker project.
+
+## Overview
+
+Microsoft dropped the support of IE11 in June 2022. So it has to be removed in Spryker.
+
+*Estimated migration time: 1h*
+
+## Update modules
+
+Run the next command to update module versions:
+```bash
+composer update spryker/zed-ui spryker-shop/shop-ui
+```
+
+## Update configuration files
+
+`.browserslistrc`:
+
+Remove IE11 usage:
+```text
+IE 11
+```
+
+`frontend/configs/development.js`: 
+
+1. Remove import of `es6-polyfill` file: 
+```js
+'es6-polyfill': es6PolyfillTs,
```

Comment:
Missed about:

- Removing the `buildVariantSettings` variable declaration and usage.
- Removing the `es6PolyfillTs` variable.

Reply:
I thought it is clear that declarations and imports should be removed )

Reply:
The migration can be done by different people and not everyone will analyze the code, but simply copy-paste it.
So it should be as detailed as possible.